vignettes/adding-support-for-reporting.Rmd

```diff
@@ -101,26 +101,26 @@ app <- init(
 if (interactive()) shinyApp(app$ui, app$server)
 ```
 
-`teal` adds another tab to the application, titled `Report previewer`. However, there is no visible change in how the module operates and appears and the user cannot add content to the report from this module. That requires inserting `teal.reporter` `ui` and `server` elements into the module body.
+`teal` adds another tab to the application, titled `Report previewer`.
+However, there is no visible change in how the module operates and appears and the user cannot add content to the report from this module.
+That requires inserting `teal.reporter` `ui` and `server` elements into the module body.
 
 ### Insert `UI` and supporting `shiny` modules for adding report cards
 
 The UI and the server logic necessary for adding cards from `example_module_with_reporting` to the report are provided by `teal.reporter::simple_reporter_ui` and `teal.reporter::simple_reporter_srv`.
 
 ```{r}
 example_module_with_reporting <- function(label = "example teal module") {
-  checkmate::assert_string(label)
   module(
     label,
     server = function(id, data, reporter) {
-      checkmate::assert_class(data, "tdata")
       moduleServer(id, function(input, output, session) {
         teal.reporter::simple_reporter_srv(
           id = "reporter",
           reporter = reporter,
           card_fun = function(card) card
         )
-        output$text <- renderPrint(data[[input$dataname]]())
+        output$text <- renderPrint(data()[[input$dataname]])
       })
     },
     ui = function(id, data) {
```

Comment:
Are we going to refactor all examples based on https://github.com/insightsengineering/teal/pull/991 ?

Reply:
:scream: Yes! I can't believe I missed it!

Reply:
I will modify all vignettes in #991 now that the module itself has been reviewed.